### PR TITLE
Finished the project

### PR DESCRIPTION
**Pull request by kjanos89**

### CODE REVIEW

File: ExerciseTracker.kjanos89/ExerciseTracker.kjanos89/Models/ExerciseDbContextFactory.cs
Comment (stevietv) [inline ExerciseTracker.kjanos89/ExerciseTracker.kjanos89/Models/ExerciseDbContextFactory.cs:1]: Tjhis class is unnecessarya dn has a duplicate connection string. The entire class can be removed

**Review by stevietv — CHANGES_REQUESTED** [review]
Hi @kjanos89 

This all looks good, you implemented the repository pattern of Controller > Service > Repository well and made use of DI to run the program. Good work.

I have one small tidy-up task for you before its merged.

**Comment by kjanos89** [comment]
Hello there, @stevietv !
Thank you for the review, removed the class as you requested.

**Comment by stevietv** [comment]
thanks for the quick tweak! approved! ✅

**Pull request merged**